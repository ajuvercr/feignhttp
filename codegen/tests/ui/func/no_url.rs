use feignhttp_codegen::get;

// error: metadata url not specified
//    |  #[get(aaa = "http://xxx")]
//    |  ^^^^^^^^^^^^^^^^^^^^^^^^^^

#[get(aaa = "http://xxx")]
fn send_get() {}

fn main() {}
