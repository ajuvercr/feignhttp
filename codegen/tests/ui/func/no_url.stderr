error: metadata url not specified
 --> tests/ui/func/no_url.rs:7:1
  |
7 | #[get(aaa = "http://xxx")]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `get` (in Nightly builds, run with -Z macro-backtrace for more info)
